lib/ansible/modules/command.py

```diff
@@ -215,10 +215,10 @@ def check_command(module, commandline):
                  'rmdir': 'state=absent', 'rm': 'state=absent', 'touch': 'state=touch'}
     commands = {'curl': 'get_url or uri', 'wget': 'get_url or uri',
                 'svn': 'subversion', 'service': 'service',
-                'mount': 'mount', 'rpm': 'yum, dnf or zypper', 'yum': 'yum', 'apt-get': 'apt',
```

Comment:
Was nuking `rpm` purposeful? I mean I'd be just as pleased if you deleted this functionality entirely.

Reply:
@sivel I nuked rpm because `zypper` isn't in base, and if someone uses an `rpm` command on a SuSE box, we shouldn't suggest they use yum/dnf there. So nuking the `rpm` one seemed like the easiest solution.